spec/features/organizer_ability_spec.rb

```diff
@@ -108,16 +108,34 @@
       visit edit_admin_conference_program_path(conference.short_title)
       expect(current_path).to eq(edit_admin_conference_program_path(conference.short_title))
 
+      # only event type exits
```

Comment:
We can keep the comment in the code anyway, please fix the typo exits -> exists and capitalize only